src/shared/report_op.c

```diff
@@ -522,7 +522,7 @@ void os_ReportdStart(report_filter *r_filter)
                     mgroup++;
                 }
 
-                free(mgroup);
+                //free(mgroup);
```

Comment:
Why is this a double free?  This is a **mgroup is a list of pointers to list char's the functions initerates thought each member then frees them when done and finally at the free's the  array of arraies.